pkg/cri/config/config_unix.go

```diff
@@ -52,6 +52,9 @@ func DefaultConfig() PluginConfig {
 
 	# CriuWorkPath is the criu work path.
 	CriuWorkPath = ""
+
+	# SystemdCgroup is default to use the systemd cgroup driver with runc
+	SystemdCgroup = true
```

Comment:
We can't make this unconditionally true when the host lacks systemd

Reply:
yes, it is difficult to use systemd in a container, If containerd is run in the container.